sharktank/tests/ops/sharded_test.py

```diff
@@ -1499,6 +1499,44 @@ def testSumBuiltinFunction(
         assert expected_result == actual_result
 
 
+class ToTest(unittest.TestCase):
```

Comment:
These pass locally but fail because of the "cuda:0". How to properly tests?

Reply:
I don't see what you could do. I don't think PyTorch support multiple logical CPU devices and we don't want to go into that rabbit hole. Maybe put a conditional skip where you check for the availability of the device.
I recall that `parameterized` supported per fixture skipping.